src/servers/ZoneServer/zoneserver.ts

```diff
@@ -1336,6 +1351,42 @@ export class ZoneServer extends EventEmitter {
     }
   }
 
+DTOhit(client: Client, packet: any) {
+    if (
+      packet.data.objectCharacterId != client.vehicle.mountedVehicle &&
```

Comment:
Why necessary checking that's the vehicle mounted by the client that send this packet?

Reply:
If you hit a non dto object Like building both characterId's belong to your vehicle, it prevents IT from being processed further